gst/nnstreamer/tensor_query/tensor_query_serversink.c

```diff
@@ -295,7 +232,8 @@ gst_tensor_query_serversink_set_caps (GstBaseSink * bsink, GstCaps * caps)
   gchar *caps_str;
 
   caps_str = gst_caps_to_string (caps);
-  gst_tensor_query_server_set_sink_caps_str (sink->server_info_h, caps_str);
+  nns_edge_set_info (sink->edge_h, "CAPS", "@query_server_sink_caps@");
```

Comment:
Offline discussion: append caps string or replace string

Reply:
Changed to replace string.